### DISCUSSION TITLE
Transfering repo ownernship

### DISCUSSION THREAD

**Discussion by Samira-El** [jmriego/pipelinewise-target-bigquery#69] [General] [answered]
Hey @jmriego, we've had an internal discussion about this repo and we agreed that it make sense to no longer have this repo under the transferwise org since we don't use it nor wish to maintain it. Thus would it be possible to transfer the ownership of this repo to yourself and manage the contributions and releases? And of course this target would remain one of the supported targets in Pipelinewise.

wdyt?

**Comment by jmriego** [comment, marked as answer]
Hi Samira!

I'd be happy to maintain this repo. The only thing I'm not sure is if I
need to create a CI/CD infrastructure myself or if it can be used the way
it's set up now. For BigQuery integrations tests I can run tests on my
account no problem.
Besides that, would I have to do something to publish to PyPi?

Also, I suppose it would be best if I still maintained as much similarity
to other targets you maintain as possible, right?

Thanks!

**Comment by Samira-El** [comment]
Hi, great to hear from you!

CI/CD pipeline uses github action and github public cloud runner, and there is no underlying infra to create, except for a Pypi user/password pair to automatically publish to Pypi upon new releases (there is a github action workflow for that).

We will also transfer ownership of the Pypi name over to you.

You don't have to keep the target similar to others, as long as it abides by the Singer spec it's all fine :)

Let me know when we can coordinate this transfer.

**Reply by jmriego** [reply]
thanks @Samira-El , that sounds perfect then.

any time now is fine by me. There's actually quite a few MR open so it would be good for me to start working on them and closing them :)

**Comment by Samira-El** [comment]
Thanks, it looks like you have a repo with the same name under your GH user: https://github.com/jmriego/pipelinewise-target-bigquery
Can you perhaps rename/delete it?

**Comment by jmriego** [comment]
sorry about that, that's the one I used before to send MR. I just deleted it

**Reply by Samira-El** [reply]
np, thanks!

**Comment by Samira-El** [comment]
I'm just sorting somethings with IT, github is not allowing me to transfer the ownership over to you.

**Comment by Samira-El** [comment]
@jmriego , I was told by IT they sent you a transfer request and it's waiting for you to accept.

**Reply by jmriego** [reply]
Thanks @Samira-El ! Yes, I received an email a few hours ago. I just accepted and should be effective any minute now

**Comment by Samira-El** [comment]
I need to transfer the ownership of the pypi name over to you as well, can you share your pypi username?

**Comment by jmriego** [comment]
Sure, it's the same one as in GitHub: `jmriego`
Thanks for all this! Do you also have to change something in the main PipelineWise repo so it uses the new pip package or does this not really effect anything on that side? I can't find anything, but just in case

**Comment by Samira-El** [comment]
You should have received an invite from pypi by now, once you accept an become owner, then you can remove the current owner (Analytics platform) from the collaborators list.

I don't think anything needs to change from PipelineWise side, as long as the pip package remains in pypi under the same name, it's all good.

you will need to set some github secrets to automate publishing of the package to pypi.
https://github.com/jmriego/pipelinewise-target-bigquery/blob/master/.github/workflows/pythonpublish.yml#L22-L23

**Comment by jmriego** [comment]
Yes, it seems like it's all working now! I was checking the pipelines and will make sure I can run the integration tests with my own account in GitHub but so far no problem at all
